src/osx/cocoa/stdpaths.mm

```diff
@@ -139,4 +139,17 @@ ConfigFileConv WXUNUSED(conv)) const
     return fn.GetFullName();
 }
 
+wxString wxStandardPaths::GetSharedLibrariesDir() const
+{
+    // Shared libraries on OSX should be stored inside the
+    // <Bundle.app>/Contents/Frameworks
+    wxFileName fn( GetExecutablePath() );
+    auto temp = fn.GetPath();
+    auto pos = temp.rfind( "/MacOS" );
+    if( pos != wxString::npos )
```

Comment:
I was thinking of using `wxFileName::RemoveLastDir()`. You seem to come up with the hardest solutions for a simple problem.

Reply:
On second thought, you probably should use a similar solution as `GetExecutablePath()`. 
https://developer.apple.com/documentation/foundation/nsbundle/1408900-builtinpluginspath

Reply:
@MaartenBent ,
> On second thought, you probably should use a similar solution as `GetExecutablePath()`. https://developer.apple.com/documentation/foundation/nsbundle/1409078-executablepath

That what OSX is using inside GetExecutablePath().

Reply:
@MaartenBent 
> I was thinking of using `wxFileName::RemoveLastDir()`. You seem to come up with the hardest solutions for a simple problem.

I used the solution from the GetInstallPrefix().

Reply:
@MaartenBent 
After thinking more about it I changed it to use your latest suggestion.

Thx.